modyn/config/schema/pipeline-schema.yaml

```diff
@@ -398,19 +407,9 @@ properties:
             - dataloader_workers
             - batch_size
             - metrics
-      device:
-        type: string
-        description: |
-          The device the model should be put on. In the future (#131), we might want this to be either "cpu" or "gpu"
-          and let the evaluator figure out the exact device, but for now, this really is the identifier of the device.
-      amp:
-        type: boolean
-        description: |
-          If True, automatic mixed precision will be used.
     required:
       - datasets
       - device
-      - amp
```

Comment:
Do we default to false if this is not given?

Reply:
Yes, exactly